docs/RELEASE.md

```diff
@@ -13,5 +13,6 @@ vx.y.z
    * Go to 'Releases' tab
    * Create new draft
    * Select tag and binaries(if  needed)
+       1. generate linux installer: `pkg -t node17-linux-x64 dist/src/index.js -o secretsfoundry-{version}-linux-x64`
```

Comment:
This is to be generated locally? If so does this need to be uploaded somewhere?

Reply:
yeah you get to upload files when drafting the release

Reply:
we shouldn't need this unless we want to maintain distribution on Github outside of npm

Reply:
We do need a binary which can be directly called. npm is not present inside argocd where this is called